Update entity API

Red-exe-Engineer opened the pull request:
I did a "little" bit of work. xD  
I removed useless code in entity.py and added new classes for sheep and tile entities.

Red-exe-Engineer committed:
Redo entity API (part 1)

Red-exe-Engineer committed:
Update entity API (part 2)

Bigjango13 commented:
Nice, could you update the README with instructions on how to use the entity class?

Red-exe-Engineer commented:
> Nice, could you update the README with instructions on how to use the entity class?

I'll try.

Red-exe-Engineer committed:
Update CmdInventory.getHeldItem()

Instead of having `{"id": str}` it now converts to `{"id": int}`, which is more intuitive.

Red-exe-Engineer committed:
Update CmdInventory

`unsafeGive` and `give` now support `minecraft.block.BLOCK`.

Red-exe-Engineer committed:
`mc.override` now supports `minecraft.block.Block`
